src/odemis/gui/cont/project.py

```diff
@@ -312,6 +316,7 @@ def create_panel(self):
         self._project_panel.add_roa_panel(self.panel)
 
         self.panel.calibration_ctrl.Bind(wx.EVT_COMBOBOX, self._on_combobox)
+        # TODO add roc 3
```

Comment:
Can you make this TODO more specific, where should roc 3 be added?

Reply:
I think I can remove this here. In the beginning I thought we also want the dropdown menu for the calibrations 3. However, there we should not give the user the possibility to select a different scintillator as this calibration is supposed to be scintillator specific.